_drafts/oryx/technology.md

```diff
@@ -0,0 +1,31 @@
+# Oryx technologies
```

Comment:
we _could_ think about adding some structure:

### Deployed stack:
- HTML
- CSS
- JS
- Web hosting
- CDN

### Dev tech
- Node (dev server)
- Typescript
- Vite
- RxJS

We should maybe mention here SSR?

What about explaining the why rather than the what as Andrii suggested?

Reply:
I know the approach to structure is opinionated, but I don't super like, that we mention JS in deployed and development stack. 

Different approach I could think about:

Technology
 - html, css
 - typescript
 - web components
 - lit
 - rxjs
 - nodejs
 - lambda
 - SSR
- vite

 Tooling & deployment:
- storybook
- netlify

Reply:
We can quickly discuss it on our sync.